com.github.Flacon.yaml

```diff
@@ -205,17 +205,12 @@ modules:
       - -DFLATPAK_BUNDLE=YES
     sources:
       - type: archive
-        url: https://github.com/flacon/flacon/archive/v5.4.0.tar.gz
-        sha256: 000eacf4528906dfe0adc1c21aedcb57bf4300e56e37150173d3a23acc8fe19f
-        # Patch is already upstream and planned for v5.4.1, this is backported to facilitate a faster flatpak release
-      - type: patch
-        path: Issue-97.patch
+        url: https://github.com/flacon/flacon/archive/v5.5.1.tar.gz
+        sha256: bdeac2268536c790ede8928156c7a35019dd05ed1b16f04238394a238950d287
 
-  # Metadata addition upstream is asked for in issue: https://github.com/flacon/flacon/issues/101
+  # Metainfo name fix is asked for upstream: https://github.com/flacon/flacon/issues/102
   - name: metadata
     buildsystem: simple
     build-commands:
-      - install -Dm 644 com.github.Flacon.appdata.xml -t /app/share/metainfo/
-    sources:
-      - type: file
-        path: com.github.Flacon.appdata.xml
+      - mv /app/share/metainfo/com.github.Flacon.metadata.xml /app/share/metainfo/com.github.Flacon.metainfo.xml
```

Comment:
Why is this still needed?

Reply:
> ```# Metainfo name fix is asked for upstream: https://github.com/flacon/flacon/issues/102```

Sokoloff accidentally gave the file the wrong name

Reply:
Oh right.